content/concepts/payment-types/partial-payments.md

```diff
@@ -8,9 +8,11 @@ labels:
 ---
 # Partial Payments
 
-In the default case, the `Amount` field of a [Payment transaction][] in the XRP Ledger specifies the exact amount to deliver, after charging for exchange rates and [transfer fees](transfer-fees.html). The "Partial Payment" flag ([`tfPartialPayment`](payment.html#payment-flags)) allows a payment to succeed by reducing the amount received instead of increasing the amount sent. Partial payments are useful for [returning payments](become-an-xrp-ledger-gateway.html#bouncing-payments) without incurring additional costs to oneself.
+The sender of any [Payment transaction][] can enable the ["Partial Payment" flag](payment.html#payment-flags) and send a payment which delivers less than the `Amount` field indicates. When processing any Payment, use the `delivered_amount` metadata field, not the `Amount` field. The `delivered_amount` is the amount a payment actually delivered.
```

Comment:
```suggestion
The sender of a [Payment transaction][] can enable the ["Partial Payment" flag](payment.html#payment-flags) to process a successful payment that delivers less than the `Amount` specified.
```

Reply:
I don't like removing references to `delivered_amount` from the first paragraph. Without that, the first reference to _the correct behavior_ is halfway down the page. (In fact, it would probably be good to have a link to that section from the reference in this first paragraph.)

Reply:
I also think it's weird to say that the sender "can enable... to process a successful payment." We should be sure not to conflate the execution of a payment that happens on-ledger with the off-ledger handling that happens as a response to a payment. From @oeggert's suggested wording, it's unclear if "process" means the on-ledger or off-ledger actions, although it's (hopefully obviously?) impossible for the sender's actions to directly control what the recipient does when they receive a payment.

Reply:
@oeggert for the reasons given by ximinez and mDuo13 above, I think your suggested sentence is less helpful than what I had proposed. Leaving this alone for now.